apps/etterlatte-saksbehandling-ui/client/src/components/person/brev/mottaker/BrevMottakerPanel.tsx

```diff
@@ -0,0 +1,295 @@
+import { AdresseType, Mottaker, MottakerType } from '~shared/types/Brev'
+import { Alert, BodyShort, Box, Heading, HStack, Loader, Tag, VStack } from '@navikt/ds-react'
+import React, { useEffect, useState } from 'react'
+import { useApiCall } from '~shared/hooks/useApiCall'
+import { getGrunnlagsAvOpplysningstype } from '~shared/api/grunnlag'
+import { Info } from '~components/behandling/soeknadsoversikt/Info'
+import { RedigerMottakerModal } from '~components/person/brev/mottaker/RedigerMottakerModal'
+import { ApiErrorAlert } from '~ErrorBoundary'
+import { mapResult } from '~shared/api/apiUtils'
+import Spinner from '~shared/Spinner'
+import { DigitalKontaktinformasjon, hentKontaktinformasjonKRR } from '~shared/api/krr'
+import { oppdaterMottaker } from '~shared/api/brev'
+import { SlettMottakerModal } from '~components/person/brev/mottaker/SlettMottakerModal'
+
+interface MottakerProps {
+  brevId: number
+  behandlingId?: string
+  sakId: number
+  mottaker: Mottaker
+  kanRedigeres: boolean
+  flereMottakere?: boolean
+  fjernMottaker?: (mottakerId: string) => void
+}
+
+const formaterMottakerType = (type: MottakerType) => {
+  switch (type) {
+    case MottakerType.HOVED:
+      return 'Hovedmottaker'
+    case MottakerType.KOPI:
+      return 'Kopimottaker'
+  }
+}
+
+export function BrevMottakerPanel({
+  brevId,
+  behandlingId,
+  sakId,
+  mottaker: initialMottaker,
+  kanRedigeres,
+  flereMottakere,
+  fjernMottaker,
+}: MottakerProps) {
+  const [mottaker, setMottaker] = useState(initialMottaker)
+
+  const [oppdaterMottakerResult, apiOppdaterMottaker] = useApiCall(oppdaterMottaker)
+  const [soeker, getSoekerFraGrunnlag] = useApiCall(getGrunnlagsAvOpplysningstype)
+  const [kontaktinfoResult, hentKontaktinfo] = useApiCall(hentKontaktinformasjonKRR)
+
+  useEffect(() => {
+    if (!behandlingId) {
```

Comment:
Skal vel aldri kunne skje det her og hvis det skjer er det bedre med et varsel?

Reply:
Joda, for det er en handling som kun skal utføres for vedtaksbrev